orm/src/main/java/org/hibernate/search/FullTextQuery.java

```diff
@@ -29,7 +28,7 @@
  * @author Hardy Ferentschik
  * @author Emmanuel Bernard
  */
-public interface FullTextQuery<R> extends Query<R>, ProjectionConstants, QueryImplementor<R> {
```

Comment:
It's a small detail really, but since there's already an issue with this commit: `QueryImplementor` already extends `org.hibernate.query.Query` (and `org.hibernate.Query`, indirectly), so we could remove the explicit ` extends org.hibernate.query.Query<R>`. What do you think?